### DISCUSSION TITLE
[EssentialsDiscord] Support /me command.

### DISCUSSION THREAD

**Discussion by codebycam** [EssentialsX/Essentials#4518] [Ideas and Feature Suggestions] [answered]
Heyo, 
Would like to suggest adding support for the `/me`(`/action`) command to the Discord module, hopefully in a similar format as displayed in-game:

![6SsjJLL](https://user-images.githubusercontent.com/41496198/132803777-4791c1b6-349e-42e8-aeba-829ca386b493.png)
![D4vQ6fv](https://user-images.githubusercontent.com/41496198/132803618-12fdee4c-1728-41da-8ab8-74e16dde1cf4.png)

Cheers ♥

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/4520
